src/main/java/ToDos.java

```diff
@@ -0,0 +1,8 @@
+public class ToDos extends Task{
+    public ToDos(String description) {
+        super(description);
+    }
+    public String getIcon(){
```

Comment:
Remember to leave a space between `()` and `{`. There are multiple instances of that throughout your code. Please remember to fix it  :)